src/components/atoms/ImageComponent/Image.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+
+import './Image.scss';
+
+
+export const ImageComponent = ({ src, size, onClick }) => {
```

Comment:
Make it pure component, and to do that do

Wrap the component with `memo`

So: 
`ImageComponent = memo({} => {})`

also to prevent onClick function from rerendering go with

`const imageOnClick = useCallback(() => onClick(), [onClick])`
`<img ... onClick={imageOnClick}`